packages/cookies-tooltip/cookies-tooltip.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import { Story, Meta } from '@storybook/react'
+
+import { Text } from '@lidofinance/text'
+import { CookiesTooltip } from './cookies-tooltip'
+
+export default {
+  title: 'CookiesTooltip/Basic',
+} as Meta
+
+export const Basic: Story = () => (
+  <>
+    <Text>
+      CookiesTooltip component have a ability to detect a user choice in other
+      tabs and third level domain.
+    </Text>
+    <Text>
+      You can check above if you:
+      <br />
+      - duplicate this tab;
```

Comment:
Better use numbers for steps, I guess: 1. 2. 3.

Reply:
ok

Reply:
done